notebooks/GWAS_blood_pressure_p2/main.py

```diff
@@ -186,7 +169,8 @@
 with herzog.Cell("python"):
     consolidated_table_name = "consolidated_metadata"
 with herzog.Cell("python"):
-    consolidate_gen3_pheno_tables(PROJECT, WORKSPACE, consolidated_table_name)
+    #consolidate_gen3_pheno_tables(PROJECT, WORKSPACE, consolidated_table_name)
```

Comment:
Why is this commented out?

Reply:
[Fixed](https://github.com/DataBiosphere/bdcat_notebooks/pull/88/commits/8b7a8c3768025c74da638cef7b55c472967e20ed). Thanks for catching this!